services/management/commands/lipas_import.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.contrib.gis.gdal import DataSource
+from django.contrib.gis.geos.collections import GeometryCollection
+
+from services.models.unit_identifier import UnitIdentifier
+
+URLS = {
+    'areas': 'http://lipas.cc.jyu.fi/geoserver/lipas/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=lipas:lipas_kaikki_alueet',
+    'paths': 'http://lipas.cc.jyu.fi/geoserver/lipas/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=lipas:lipas_kaikki_reitit'
+}
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        logger.info('Retrieving all external unit identifiers from the database...')
+
+        # Get all external datasource records from the database
+        unit_identifiers = UnitIdentifier.objects.filter(namespace='lipas')
+
+        # Build a lipas_id-indexed dictionary of units
+        units_by_lipas_id = {}
+        for unit_identifier in unit_identifiers:
+            units_by_lipas_id[int(unit_identifier.value)] = unit_identifier.unit
+
+        # Get path and area data from the Lipas WFS
+        logger.info('Done.')
+        logger.info('Retrieving geodata from Lipas...')
+        ds_paths = DataSource(URLS['paths'])
```

Comment:
Because the LIPAS docs explicitly asks us to restrict our data retrievals, I could be nice to figure out filters so that only the relevant data is fetched:
 - a geographical boundary for the metropolitan area (configurable) and/or
 - a municipality filter (if there is a municipality field)
 - a sport category filter